tslearn/metrics/dtw_variants.py

```diff
@@ -190,7 +190,11 @@ def dtw_path(s1, s2, global_constraint=None, sakoe_chiba_radius=None,
     s2 = to_time_series(s2, remove_nans=True)
 
     if len(s1) == 0 or len(s2) == 0:
-        raise ValueError("One of the input time series contains only nans or has zero length.")
+        raise ValueError(
+            "One of the input time series contains only nans or has zero length.")
+
+    if s1.shape[1] != s2.shape[1]:
+        raise ValueError('dimension mismatch')
```

Comment:
Sorry that it took me so long to answer...

Could you provide an error message that would be more informative?

Reply:
Thank you for your reply.

Ok, I'll fix it.